tests/ut/azure_iot_provisioning_client_ut.c

```diff
@@ -868,8 +946,11 @@ uint32_t ulGetAllTests()
         cmocka_unit_test( testAzureIoTProvisioningClient_Register_SubscribeAckFailure ),
         cmocka_unit_test( testAzureIoTProvisioningClient_Register_PublishFailure ),
         cmocka_unit_test( testAzureIoTProvisioningClient_Register_RegistrationFailure ),
+        cmocka_unit_test( testAzureIoTProvisioningClient_Register_QueryPublishFailure ),
+        cmocka_unit_test( testAzureIoTProvisioningClient_Register_QueryShortFailure ),
         cmocka_unit_test( testAzureIoTProvisioningClient_Register_QueryFailure ),
-        cmocka_unit_test( testAzureIoTProvisioningClient_Register_InvalidResponseFailure ),
+        cmocka_unit_test( testAzureIoTProvisioningClient_Register_QueryDeviceDisabledResponse ),
```

Comment:
is this Failure or Success?
```suggestion
        cmocka_unit_test( testAzureIoTProvisioningClient_Register_QueryDeviceDisabledResponseFailure ),
```

Reply:
Fixed.